environment.yml

```diff
@@ -0,0 +1,35 @@
+# Usage: `conda env update -f environment.yml`
```

Comment:
what is the reason for adding `conda` env here?

Reply:
don't pay attention to this, it's because this PR is based on #849

Reply:
good to know lol :]